websites/G/GenoPad/dist/metadata.json

```diff
@@ -0,0 +1,25 @@
+{
+  "$schema": "https://schemas.premid.app/metadata/1.3",
+  "service": "GenoPad",
+  "author": {
+    "name": "Deleted 7.12i",
+    "id": "591702135310385157"
+  },
+  "category": "other",
+  "logo": "https://i.imgur.com/QAwnyL4.png",
+  "thumbnail": "https://i.imgur.com/Tp42TcS.png",
```

Comment:
no sure if that thumbnail is really a "Promotional banner or a screenshot of the website homepage"